types/sparql-http-client/index.d.ts

```diff
@@ -23,6 +23,7 @@ export {
 export interface QueryOptions {
     headers?: HeadersInit;
     operation?: "get" | "postUrlencoded" | "postDirect";
+    parameters?: { [key: string]: undefined | string | string[] };
```

Comment:
I think this would be cleaner

```suggestion
    parameters?: Record<string, string | string[] | undefined>;
```